alerts/src/iosMain/kotlin/AlertPresenter.kt

```diff
@@ -143,7 +153,11 @@ actual class AlertPresenter(
                 }
             }
         }.run {
-            parent.presentViewController(this, animated, completion)
+            logger.info(TAG, "Displaying alert dialog with title: ${alert.title}")
+            parent.presentViewController(this, animated) {
+                logger.info(TAG, "Dismissing alert dialog with title: ${alert.title}")
```

Comment:
Completion of presentViewController does not mean the alert is dismissed. It means that the Alert is now being presented (i.e. the animation has completed)

Reply:
Yes indeed (should have read the docs, lucky you knew that). Adjusted to take the event from Action click event for dismissing.